proposals/lip-0014.md

```diff
@@ -366,6 +366,8 @@ If any of the validations fail, we discard the block. We further ban the peer se
 
 *   Check if the `B.heightPrevoted` value in the block header is correct. This is outlined in the section "Validating Additional Block Header Properties" above.
 *   Check if there is any contradiction between the block header and any block by the same delegate in the current chain. For performance reasons, for a block `B` that is supposed to be added to the tip of the chain, we only check the blocks at heights `B.height-303` up to `B.height-1`. This verification step is formally defined in the section "Detecting Contradicting Block Headers" above.
+Note that if there is no block by the same delegate at heights `B.height-303` up to `B.height-1`, there is no contradiction.
```

Comment:
As discussed, the formulation `there is not contradiction` should be changed to something like "there is no contraction with any non-finalized block"

Reply:
Changed to  `there is no contradiction with any block at these heights.`.